lib/cadet_web/controllers/grading_controller.ex

```diff
@@ -99,6 +117,21 @@ defmodule CadetWeb.GradingController do
     response(401, "Unauthorised")
   end
 
+  swagger_path :unsubmit do
+    post("/grading/{submissionId}/unsubmit")
+    summary("Unsubmit submission for an assessment by Avenger of student.")
```

Comment:
Make it clearer: `Unsubmit submission. Can only be done by the Avenger of a student.`